src/Aaron/PasswordValidation.java

```diff
@@ -0,0 +1,73 @@
+package Aaron;
+
+public class PasswordValidation {
+    /*String -- Password Validation Task
+    Write a return method that can verify if a password is valid or not.
+    Requirements:
+    1. Password MUST be at least have 6 characters and should not contain space
+    2. PassWord should at least contain one upper case letter
+    3. PassWord should at least contain one lowercase letter
+    4. Password should at least contain one special characters
+    5. Password should at least contain a digit
+    if all requirements above are met, the method returns true, otherwise returns  false*/
+
+    public static void main(String[] args) {
+        System.out.println(PassWordvalidation("Ab*1Ab"));
+        System.out.println(PassWordvalidation("Ab*1A "));
+        System.out.println(isValidPassword("Ab*1Ab"));
+        System.out.println(isValidPassword("AC*1AA"));
+    }
+
+    public static boolean PassWordvalidation(String password) {
+
+        String lowercase = "(.*[a-z].*)";
+        String uppercase = "(.*[A-Z].*)";
+        String numbers = "(.*[0-9].*)";
+        String specialchars = "(.*[ -/, :-@].*)";
+
+        boolean HasLower = password.matches(lowercase),
+                HasUppere = password.matches(uppercase),
+                HasDigits = password.matches(numbers),
+                HasSpecial = password.matches(specialchars),
+                Valid = false;
+
+        if (password.length() >= 6 && !password.contains(" "))
+            if (HasLower && HasUppere && HasDigits && HasSpecial)
```

Comment:
If the variable name will be changed in the line 29, make sure to change it here as well =)